### ISSUE TITLE
ngbModal - autofocus for button works only once

### ISSUE THREAD

**Issue by 730alchemy** [ng-bootstrap/ng-bootstrap#2109]
### Bug description:
When a `button` element with the `autofocus` attribute is included in the Angular template, the button has focus only the first time the modal dialog is opened.  When the dialog is opened again, the button does not have focus.

### Link to minimally-working plunker that reproduces the issue:
https://plnkr.co/FGB04wHwoRAUI9jOx3BQ

### Version of Angular, ng-bootstrap, and Bootstrap:

Angular: 2.3.1

ng-bootstrap: 1.0.0-alpha.20

Bootstrap: 4.0.0-alpha.6

**Comment by 730alchemy** [comment]
The same problem exists with latest versions:

Plunker: https://embed.plnkr.co/oN6e7OmS5nbtCTCokAF0/

Angular: 5.0.2

ng-bootstrap: ng-bootstrap@1.0.0-beta.9

Bootstrap: 4.0.0-beta.3

**Comment by pkozlowski-opensource** [comment]
Duplicate of #938. Please note that browsers' behaviour for dynamically inserted content with the `autofocus` attribute is inconsistent. Anyway, let's centralise the discussion in #938.
